ddtrace/api.py

```diff
@@ -25,7 +26,13 @@ def __init__(self, hostname, port, headers=None, encoder=None):
 
         # overwrite the Content-type with the one chosen in the Encoder
         self._headers = headers or {}
-        self._headers.update({'Content-Type': self._encoder.content_type})
+        self._headers.update({
+            'Content-Type': self._encoder.content_type,
+            'Datadog-Meta-Lang': 'python',
```

Comment:
@gabsn I'd say to add that string (`'python'`) as a new service metadata. So the service when it's reported once, knows that `my-distributed-intake` is a Python application.